permissions/permissions-beta.json

```diff
@@ -55502,7 +55502,7 @@
     "DelegatedPersonal": [
      "Not supported."
     ],
-    "Application": [
+    "Application": 
```

Comment:
Hi @gracet9182 !

This change will break the permissions-beta JSON, as you removed the opening `[` but kept the closing `]`